settings.json

```
@@ -0,0 +1,600 @@
+{
+    "type": "object",
+    "properties": {
+        "ALLOWED_HOSTS": {
+            "type": "array",
+            "default": []
+        },
+        "DATABASES": {
+            "type": "object",
+            "properties": {
+                "default": {
+                    "type": "object",
+                    "properties": {
+                        "NAME": {
+                            "type": "string",
+                            "default": "nautobot"
+                        },
+                        "USER": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "PASSWORD": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "HOST": {
+                            "type": "string",
+                            "default": "localhost"
+                        },
+                        "PORT": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "CONN_MAX_AGE": {
+                            "type": "integer",
+                            "default": 300
+                        },
+                        "ENGINE": {
+                            "type": "string",
+                            "default": "django.db.backends.postgresql"
+                        }
+                    }
+                }
+            }
+        },
+        "CACHES": {
+            "type": "object",
+            "properties": {
+                "default": {
+                    "type": "object",
+                    "properties": {
+                        "BACKEND": {
+                            "type": "string",
+                            "default": "django_redis.cache.RedisCache89"
```

```suggestion
                            "default": "django_redis.cache.RedisCache"
```

Here too should we specify the options as either `django_redis.cache.RedisCache` or `django_prometheus.cache.backends.redis.RedisCache`?

---

Just to confirm, something like this?
```
"BACKEND": {
      "type": "string",
      "default": "[django_redis.cache|django_prometheus.cache.backends.redis].RedisCache"
},
```

---

No, the default needs to be a specific value - I was meaning something like https://json-schema.org/understanding-json-schema/reference/enum#enumerated-values